### ISSUE TITLE
Elastic 2.0 support

### ISSUE THREAD

**Issue by r--w** [belogik/goes#39]
Hi, 
Do you have any plans for that?

**Comment by jeromer** [comment]
No. To be honnest I really do not have time to maintain this library and I would be happy if someone could take over this project.

See #40
